src/messages/embed.ts

```diff
@@ -0,0 +1,260 @@
+import type {
+	APIEmbed,
+	APIEmbedAuthor,
+	APIEmbedField,
+	APIEmbedFooter,
+	APIEmbedImage,
+	APIEmbedProvider,
+	APIEmbedThumbnail,
+	APIEmbedVideo,
+} from 'discord-api-types/v8';
+
+/**
+ * Represents an embed in a message (image/video preview, rich embed, etc.)
+ */
+export class Embed implements APIEmbed {
+	/**
+	 * An array of Fields of this Embed.
```

Comment:
```suggestion
	 * An array of fields of this embed.
```

Reply:
Also.. do we want to keep the `.` on property descriptions? I don't think I used those in SlashCommands? @discordjs/the-big-3

Reply:
I personally use `.`, and I did for [formatters](https://github.com/discordjs/builders/blob/main/src/messages/formatters.ts).